app/coffee/Features/Editor/EditorHttpController.coffee

```diff
@@ -29,17 +30,17 @@ module.exports = EditorHttpController =
 		ProjectGetter.getProjectWithoutDocLines project_id, (error, project) ->
 			return callback(error) if error?
 			return callback(new Error("not found")) if !project?
-			ProjectGetter.populateProjectWithUsers project, (error, project) ->
+			CollaboratorsHandler.getMembersWithPrivilegeLevels project, (error, members) ->
 				return callback(error) if error?
 				UserGetter.getUser user_id, { isAdmin: true }, (error, user) ->
 					return callback(error) if error?
-					AuthorizationManager.getPrivilegeLevelForProject project, user, (error, canAccess, privilegeLevel) ->
+					AuthorizationManager.getPrivilegeLevelForProject user_id, project_id, (error, privilegeLevel) ->
 						return callback(error) if error?
-						if !canAccess
+						if !privilegeLevel
```

Comment:
having false as a privilege level feels a bit confusing at times like this. A couple of other options could be

    if privilegeLevel == PrivilegeLevels.noAccess

-

    if hasAccess(privilegeLevel)

or maybe even

    if privilegeLevel == false

as this last one is a bit more verbose about what is it doing. The !privilegeLevel is completely correct but doesn't feel obvious. Im waiting to see someone write code in 2 years with

    if privilegeLevel?
        theyHaveAccess()

One to debate.